app/services/notifications/new_comment/send.rb

```diff
@@ -1,7 +1,12 @@
+# frozen_string_literal: true
+#
 # send notifications about the new comment
+
 module Notifications
   module NewComment
     class Send
+      attr_reader :comment
```

Comment:
Do we really need a public `attr_reader`?

Reply:
Maybe not. I can add `private :comment`

Reply:
This line was under `private` section, so I was wondering why was it moved.

Reply:
Yeah, missed it. Just more readable

Reply:
@araslanov-e let me know when you changed this, I'll then approve the PR

Reply:
@rhymes done